src/library/scanner/libraryscanner.h

```diff
@@ -55,13 +55,11 @@ class LibraryScanner : public QThread {
                    ConfigObject<ConfigValue>* pConfig);
     virtual ~LibraryScanner();
 
+  public slots:
     // Call from any thread to start a scan. Does nothing if a scan is already
     // in progress.
     void scan();
```

Comment:
It didn't -- previously the QAction was hooked to a slot in MixxxMainWindow and that slot called the LibraryScanner directly. Now WMainMenuBar is directly connected to LibraryScanner.